src/components/auth0NavBar/Auth0Profile.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { useAuth0 } from "@auth0/auth0-react";
+
+const Auth0Profile = () => {
+  const { user, isAuthenticated, getAccessTokenSilently } = useAuth0();
+
+  /**
+   * Allows us to fetch the accessToken that we will need for git gateway.
+   */
+  const callApi = async () => {
+    const domain = "dev-fmjy7n5n.us.auth0.com";
```

Comment:
Domain that is automatically generated for my Single Page Application within Auth0.

Reply:
we can move this component to the top nav bar